en/integrations/plugin-packs/procedures/applications-monitoring-centreon-mbi.md

```diff
@@ -3,122 +3,329 @@ id: applications-monitoring-centreon-mbi
+(EMS), providing you with full visibility of your infrastructures and 
+application activities through ITIL compliant reporting.
+
+## Plugin-Pack assests
+
+### Monitored objects
+
+* Centreon MBI reporting server
+
+### Collected metrics
+
+In addition to modes and metrics described here, it is also possible to monitor 
+the following indicators:
+
+* DWH-db-content: Check if the database is up-to-date
+* DWH-partitions: Check if all MySQL partitions are up-to-date
+* failed-jobs: Check if any job failed
+* Ntp: Check time offset of server with ntp server.
+* proc-cbis: State of the cbis process on the Centreon MBI reporting
+server
+* proc-mysql: State of the MySQL process on the Centreon MBI reporting
+server
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cpu-->
+
+| Metric name                        | Description                                   |
+| :--------------------------------- | :-------------------------------------------- |
+| cpu.utilization.percentage         | CPU utilization. Unit : %                     |
+| core.cpu.utilization.percentage    | Per Core CPU utilization. Unit : %            |
+
+<!--Memory-->
+
+| Metric name             | Description                                              |
+| :---------------------  | :------------------------------------------------------- |
+| memory.usage.bytes      | Memory usage on the device. Unit : Bytes                 |
+| memory.free.bytes       | Free memory on the device. Unit : Bytes                  |
+| memory.usage.percentage | Percentage of Memory usage on the device. Unit : %       |
+| memory.buffer.bytes     | Buffered Memory allocation. Unit : Bytes                 |
+| memory.cached.bytes     | Cached Memory allocation. Unit : Bytes                   |
+| memory.shared.bytes     | Shared Memory allocation. Unit : Bytes                   |
+
+<!--Swap-->
+
+| Metric name                 | Description                                       |
+| :-------------------------- | :------------------------------------------------ |
+| swap.usage.bytes            | Used swap. Unit: Bytes                            |
+| swap.free.bytes             | Free swap. Unit: Bytes                            |
+| swap.usage.percentage       | Percentage of used swap. Unit: %                  |
+
+<!--Load-->
+
+| Metric name                 | Description                                        |
+| :-------------------------- | :------------------------------------------------- |
+| load1                       | System load 1 minute-sample                        |
+| load5                       | System load 5 minutes-sample                       |
+| load15                      | System load 15 minutes-sample                      |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
 
 ## Prerequisites
 
-### Centreon Plugin
+To monitor a Centreon MBI reporting server, the NRPE3 and SNMP services must be
+installed and configured.
+
+### NRPE3
```

Comment:
Maybe for the NRPE3, simply add link to the NRPE3 monitoring procedure and just add the part :
```
Futhermore, add the following lines at the end of the file:

command[check_partitions]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --partitions
command[check_db]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --db-content
command[check_jobs]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --jobs
```